vizro-core/docs/pages/user-guides/card-button.md

```diff
@@ -409,7 +409,7 @@ To create a navigation card, do the following:
 
 - Insert the [`Card`][vizro.models.Card] into the `components` argument of the [`Page`][vizro.models.Page]
 - Provide the `text` parameter with a title and some description
-- Provide the `href` parameter (relative or absolute URL)
+- Provide the `href` parameter (relative or absolute URL). If it refers to a `Page` inside the `Dashboard`, you can insert the `Page.title` directly.
```

Comment:
Here I think it does matter that we are correct. I'd leave as `Page.title` as the first thing you mention, since 99% of the time that's correct and easier to understand for a user than `Page.id`. But then add something afterwards saying that technically it's `Page.id` and link to https://vizro.readthedocs.io/en/latest/pages/user-guides/pages/#customizing-the-page-url.